promql/parse_test.go

```diff
@@ -1333,12 +1332,6 @@ var testStatement = []struct {
 	}, {
 		input: `foo{a!~"b"} = bar`,
 		fail:  true,
-	}, {
-		input: `ALERT SomeName IF time() LABELS {} 
-			SUMMARY "Global request rate low"
-			DESCRIPTION "The global request rate is low"
-		`,
-		fail: true,
```

Comment:
There was no clear point to this test to begin with. Deletion wasn't due to this change.